### PR TITLE
 Cleanup secho

### PR DESCRIPTION
**Pull request by oz123**
Working on reusing rich from pipenv, so we can reuse vendored libs from it.

### CODE REVIEW

**Comment by matteius** [comment]
Does rich already handle the NO_COLOR env var?

**Comment by oz123** [comment]
Yes. It does:

https://rich.readthedocs.io/en/stable/console.html?highlight=NO_COLOR#environment-variables

File: pipenv/utils/pip.py [hunk 1]
```diff
@@ -174,20 +165,12 @@ def pip_install_deps(
         pip_command.extend(["-r", normalize_path(file.name)])
         if project.s.is_verbose():
             msg = f"Install Phase: {'Standard Requirements' if file == standard_requirements else 'Editable Requirements'}"
-            click.echo(
-                click.style(msg, bold=True),
-                err=True,
-            )
+            err.print(msg, style="bold")
```
Comment (matteius) [inline pipenv/utils/pip.py:168]: Is this conversion 1:1?

Comment (oz123) [reply pipenv/utils/pip.py:168]: Yes. The API of rich is easy to get.

File: pipenv/routines/clean.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 from pipenv.patched.pip._internal.build_env import get_runnable_pip
 from pipenv.routines.lock import do_lock
+from pipenv.utils import console, err
```
Comment (matteius) [inline pipenv/routines/clean.py:5]: But where does this import come from?

Comment (oz123) [reply pipenv/routines/clean.py:5]: I initialized them here:
https://github.com/pypa/pipenv/blob/8240ea73d2e348960f97ca01e7ed0f169876fb0c/pipenv/utils/__init__.py

**Review by matteius — APPROVED** [review]

**Pull request merged**